src/components/MiniMovie/MiniMovie.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import './MiniMovie.css'
+
+class MiniMovie extends Component {
+    constructor() {
+        super()
+        this.state = {
+            // potato: false this is an unnecessary potato, should this be a functional component
+        }
+    }
+
+    handleHover = event => {
+        console.log('hi')
+        // this runs 3 times each hover for some reason
+        this.setState(prevState => {
+            return {
+                isHoveredOn: !prevState.isHoveredOn
+            }
+        })
+    }
+    
+    render(){
+        const {movie} = this.props
+        // same thing ^^ const movie = props.movie ^^
+        const {id, poster_path, backdrop_path, title, average_rating, release_date} = movie
+        const {handleMovieView} = this.props
```

Comment:
Love seeing you reach for destructuring!  Could these three lines be streamlined?